SubscriptionService/SubscriptionService.BusinessLogic/Mapping/SubscriptionMappingProfile.cs

```diff
@@ -0,0 +1,22 @@
+﻿using AutoMapper;
+using SubscriptionService.BusinessLogic.Extensions;
+using SubscriptionService.BusinessLogic.Models.Subscription;
+using SubscriptionService.BusinessLogic.Models.TariffPlan;
+using SubscriptionService.DataAccess.Entities;
+
+namespace SubscriptionService.BusinessLogic.Mapping
+{
+    public class SubscriptionMappingProfile : Profile
+    {
+        public SubscriptionMappingProfile()
+        {
+            CreateMap<Subscription, GetSubscriptionDto>()
+                .ForMember(dest => dest.TariffPlanName, options => options.MapFrom(src => src.TariffPlan.Name));
```

Comment:
its good practice to always write down manually what property of one model you map with property of another model, because you can change property name in one model and forget to change it in another. and your mapping will no longer work)